src/env.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="vite/client" />
```

Comment:
Read it from the docs: https://vitejs.dev/guide/env-and-mode.html#intellisense-for-typescript